doc/specs/GitHub-discussions-spec.md

```diff
@@ -0,0 +1,57 @@
+# GitHub discussions spec
+
+## What is GitHub Discussions
+
+GitHub Discussions is a feature that currently is in closed beta status. It will be released until the end of the year. 
+
+It's a new tab in the GitHub repo page where people can start new threads like in a forum.
+
+The threads must be added to a category. The answer feature can be enabled or disabled for categorys. If enabled a person with triage permission or the creator of the thread can mark a reply as answer.
+
+You can sort threads in the overview for categories and for (un-)answered.
+
+One thread can be pinned on the main page of the discussion section. (One with rules or something like that).
+
+Threads uses the same numeration like PR and issues. So they can be linked by the others. A conversation can also be locked like in issues and PR.
+
+## Why do we need this for PowerToys
+
+1. Sometimes people just want to say thank you to the PT team (like in https://github.com/microsoft/PowerToys/issues/7873) and a thread in GitHub Discussions where every one can comment if he just want to say thank you, is much better for the users and the team than open every time a new issue for that.
+
+2. Questions: If people just have questions, it gets difficult by asking them in issues. It's more useful for both sides (team and user) when questions are asked in a different section. Because in GitHub discussion you can mark a reply as the answer and other users can find the answer quicker.
+
+3. Off-topic. Sometimes issue discussions get very and sometimes a little off-topic (like https://github.com/microsoft/PowerToys/issues/7916#issuecomment-724813783) And it would be better if you can open a thread and write "As seen in #XXXX I think the real reason for ... is ..."
+
+
+## Settings
+
+📝General (answers not enabled)
+❓Q&A (answers enabled)
+(💬Off-topic (answers not enabled))
+
+Pinned thread: (conversation locked)
+```
+
+# Hello and welcome to our GitHub discussion section.
+
+Here you can ask questions and say thank you.
+
+
+[Some links to other important threads]
+```
+
+Other threads:
+-	A thread where you can say thank you to the PT team.
+-	
+
+
+## What when it don't work
+
+If the community denies the feature, it gets too much work, it gets used very low or if it gets spammed we can simply deactivate the feature.
```

Comment:
gets filled with spam? or something like that

Reply:
deactivating stuff comes at a cost.  Turning stuff on / off is actually rather expensive in terms of redirecting and making sure people know what / where to go.  This is why i'm really pushing for the 'why'